### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rishabhKalakoti**
No form link specified.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,6 @@ Pre-requisite
 
 * knowledge of GitHub, not a must though you can learn basics in a couple of hours.
 * Presence on Telegram Group (install telegram first if an error or join from the desktop )
-* Fill this form with SOP(statement of purpose)
```
Comment (theSage21) [inline README.md]: if you have it why don't you add the form link. @Shivank98 I recollect you saying this was a requirement?

Comment (Shivank98) [reply README.md]: Yeah, it was but now we can do it like any other Open Source project.

Comment (Shivank98) [reply README.md]: looks go to me.

Comment (rishabhKalakoti) [reply README.md]: @theSage21 , please merge this one too.

**Pull request merged**